ibm/config.go

```diff
@@ -1097,6 +1106,29 @@ func (c *Config) ClientSession() (interface{}, error) {
 		}
 	}
 
+	// Construct an "options" struct for creating the service client.
+	atrackerClientURL, err := atrackerv1.GetServiceURLForRegion("private." + c.Region)
```

Comment:
Here by default the atracker is supported by private endpoints. We recently introduced the visibility parameter at provider level. By default its public...So if c.visibility == public may be can we set the error to session.atrackerClientErr  = Public endpoints is  not supported by this service

Reply:
This will be a hand edit.